compiler/src/codegen/compcore.re

```diff
@@ -33,6 +34,13 @@ let gensym_label = s => {
 };
 let reset_labels = () => gensym_counter := 0;
 
+let needs_exceptions = ref(false);
```

Comment:
I'm not the biggest fan of this approach, but it was definitely the simplest. The alternatives I thought of were another compilation mode specifically for exceptions, and registering imports in `compcore.re` as they're used (so not just for exceptions, but for everything). I think the latter would be better than this, but it'd add quite a bit of complexity to implement it. If folks really don't like how I did it though, I can explore doing it that way.

Reply:
I don't think it should block this PR, but I think that we should open an issue to take the latter approach. Maybe also add a comment explaining why it's here? I feel like it could become a mystery variable very quickly (at least to me).

Reply:
Added a comment 👍